### PR TITLE
Clarify relationship with SMPTE-TT

### LINKED ISSUE #312: Improve text of I.4 SMPTE-TT (SMPTE ST 2052-1)
The appendix does not take into account that IMSC1.1 is based on TTML2, whereas SMPTE ST 2052-1 is based on TTML1.

### PR DESCRIPTION
**Pull request by palemieux**
Closes #312

### CODE REVIEW

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -4363,8 +4363,8 @@ <h3>SMPTE-TT (SMPTE ST 2052-1)</h3>
       <p>[[!SMPTE2052-1]] specifies the use of the DFXP Full Profile (see Appendix F.3 at [[!TTML2]]) supplemented by a number of
       extensions, including <code>http://www.smpte-ra.org/schemas/2052-1/2010/smpte-tt#image</code>.</p>
 
-      <p>This specification defines practical constraints on [[!SMPTE2052-1]], supplemented by a few extensions defined at <a href=
-      "#features-and-extensions"></a>. These constraints and extensions are intended to reflect industry practice.</p>
+      <p>This specification defines practical constraints on [[!SMPTE2052-1]], supplemented by extensions defined herein and
+      selected TTML2 features. These constraints and extensions are intended to reflect industry practice.</p>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: s/`TTML2`/`[[!TTML2]]` ?

**Comment by nigelmegitt** [comment]
@palemieux how did you merge this with no positive reviews?

**Comment by nigelmegitt** [comment]
@palemieux Sorry, I see, you weren't merging into a protected branch. Not to worry.

**Comment by palemieux** [comment]
@nigelmegitt Oh. I saw your review, and assumed you had approved. My mistake.

**Comment by nigelmegitt** [comment]
@palemieux I hadn't, but it's fine, and if it hadn't been merged I would have approved it.

**Comment by nigelmegitt** [comment]
(of course I clearly forgot about this and failed to approve it, so that last comment was rather contingent on me somehow realising there was action to take!)

**Pull request merged**